lib/graphql/define/defined_object_proxy.rb

```diff
@@ -10,6 +10,11 @@ def types
         GraphQL::Define::TypeDefiner.instance
       end
 
+      # TODO: do I actually want to add this?
```

Comment:
:+1: I [moved it to `spec_helper.rb`](https://github.com/rmosolgo/graphql-ruby/pull/300/commits/7b91e3ff109369bd681f7e7b05959eace0fc0007#diff-93830fa29d616f7c87903d08b5b1b29aR22) so it's there for testing but not default for the gem :)